app/controllers/password_resets_controller.rb

```diff
@@ -9,10 +9,10 @@ def edit
   end
 
   def create
-    @user = User.find_by(email: params[:password_reset][:email].downcase)
-    if @user
-      @user.create_reset_digest
-      @user.send_password_reset_email
+    result =
```

Comment:
@Enriikke I'm sticking to `result` as the variable name in some of these. Is that convention or should I be a little more critical in my naming?